### DISCUSSION TITLE
Search Results - Issue - Localization - Support search page and site collection level termset in different site collections

### DISCUSSION THREAD

**Discussion by EMaarten** [microsoft-search/pnp-modern-search#3220] [Ideas]
Hi,

I'm trying to configure a search results web part with localized values for a managed metadata field. This managed metadata field is connected to a termset at site collection level. The localized results shown by the search results web part are perfectly fine on the site containing the data and termset by using the auto-property. 

When trying to configure the web part with the same configuration at another site collection, the auto-property isn't available. In the developer toolbar I noticed an HTTP call to retrieve the term labels but targeting the current site collection (the one containing the search page, not the one containing the data and the termset). As the termset doesn't exist in this site collection, the translation fails and it remains empty.

So could it be that localization doesn't work when the terms are stored in a termset at site collection level while the search page is stored at another site collection? (It does work when the page and termset are stored in the same site collection.)

**Comment by wobba** [comment]
The API calls are made towards the endpoint you are at to ensure the API works. I know term lookup works for global terms, but never tried site collection ones across site collection, so likely not a supported scenario if it fails. I'll tag this as something for someone to look into if it's possible to somehow support.

**Reply by EMaarten** [reply]
Thank you!
